bitnami/kong/README.md

```diff
@@ -0,0 +1,326 @@
+| `ingressControllerImage.repository`                     | kong ingress controller image name                                                                                     | `bitnami/kong`                                       |
+| `ingressControllerImage.tag`                            | kong ingress controller image tag                                                                                      | `{TAG_NAME}`                                            |
+| `ingressControllerImage.pullPolicy`                     | kong ingress controller image pull policy                                                                              | `IfNotPresent`                                          |
+| `ingressControllerImage.pullSecrets`                    | Specify docker-registry secret names as an array                                                       | `[]` (does not add image pull secrets to deployed pods) |
+| `migrationImage.registry`                       | kong migration job image registry                                                                                 | `docker.io`                                             |
+| `migrationImage.repository`                     | kong migration job image name                                                                                     | `bitnami/kong`                                       |
+| `migrationImage.tag`                            | kong migration job image tag                                                                                      | `{TAG_NAME}`                                            |
+| `migrationImage.pullPolicy`                     | kong migration job image pull policy                                                                              | `IfNotPresent`                                          |
+| `migrationImage.pullSecrets`                    | Specify docker-registry secret names as an array                                                       | `[]` (does not add image pull secrets to deployed pods) |
+| `replicaCount`                         | Number of replicas of the kong Pod                                                                  | `2`                                                     |
+| `updateStrategy`                       | Update strategy for deployment                                                                         | `{type: "RollingUpdate"}`                               |
+| `schedulerName`                        | Alternative scheduler                                                                                  | `nil`                                                   |
+| `database`                        | Select which database backend Kong will use. Can be 'postgresql' or 'cassandra'                                                                                  | `postgresql`                                                   |
```

Comment:
I guess this parameter should be under "Kong Container Parameter"

Reply:
It is used by the migration deployment as well

Reply:
I tried not using this parameter and rely on the cassandra.enabled, postgresql.enabled, cassandra.external or postgresql.enabled but it was a bit messy